vault/resource_secrets_sync_config.go

```diff
@@ -9,13 +9,14 @@ import (
 
 	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
 	"github.com/hashicorp/terraform-provider-vault/internal/provider"
-	"github.com/hashicorp/vault/helper/namespace"
 )
 
 const (
 	fieldDisabled      = "disabled"
 	fieldQueueCapacity = "queue_capacity"
+	rootNamespaceID    = "root"
```

Comment:
nit: This is available in the Vault sdk at https://github.com/hashicorp/vault/blob/fab28995080b58e7e8a20ba382b15a594368a2ac/sdk/helper/testhelpers/namespaces/namespaces.go#L22

Reply:
Thanks! Updated to use the sdk constant